### PR TITLE
:memo: Create introspector tutorial

### PR DESCRIPTION
**Pull request by quilicicf**

### CODE REVIEW

File: introspector/src/main/java/Contact.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+public class Contact {
```
Comment (thboileau) [inline introspector/src/main/java/Contact.java]: I wonder if we should add copyright header.

File: introspector/introspecter.iml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (thboileau) [inline introspector/introspecter.iml]: I guess it should not be versionned.

Comment (quilicicf) [reply introspector/introspecter.iml]: Indeed.

File: introspector/src/main/java/Contact.java (hunk 1 above)
Comment (quilicicf) [reply introspector/src/main/java/Contact.java]: Asked in an email. I'll come back to that when Jérôme answers.

**Pull request merged**